.github/workflows/integration-tests.yml

```diff
@@ -64,10 +64,10 @@ jobs:
       - name: Install Composer dependencies
         uses: ramsey/composer-install@v1
         with:
-          composer-options: "${{ matrix.composer-options }}"
+          composer-options: '${{ matrix.composer-options }}'
 
-#      - name: Run unit tests
-#        run: composer unit
+      #      - name: Run unit tests
+      #        run: composer unit
```

Comment:
```suggestion
```

Let's just take this out for now. Easy enough to recreate if/once tests are split.